app/services/qa/linked_data/ldpath_service.rb

```diff
@@ -6,6 +6,9 @@ module LinkedData
     class LdpathService
       VALUE_ON_ERROR = [].freeze
 
+      class_attribute :predefined_prefixes
+      self.predefined_prefixes = Ldpath::Transform.default_prefixes.with_indifferent_access
```

Comment:
Hold a copy of the default prefixes defined in the ldpath gem so we can use them locally.